terminal.py

```diff
@@ -92,9 +92,12 @@ def main():
     menu_parser.add_argument("opt", choices=choices)
     completer = NestedCompleter.from_nested_dict({c: None for c in choices})
 
-    if os.name == "nt":
-        sys.stdin.reconfigure(encoding="utf-8")
-        sys.stdout.reconfigure(encoding="utf-8")
+    try:
```

Comment:
@aia is it ok to add a "try-catch" here ?

At least 6 people now had to comment these lines to have the terminal working. Not sure what else we can do

Reply:
Try is fine here. Especially if a lot of people are complaining.

Reply:
Windows and UTF-8 behavior is... interesting.